mono/mini/mini.c

```diff
@@ -4316,10 +4316,7 @@ mini_get_cpu_features (MonoCompile* cfg)
 #endif
 	}
 #endif
-	MonoCPUFeatures features_ = features;
-
+	
 	// apply parameters passed via -mattr
-	features = (MonoCPUFeatures) (features | mono_cpu_features_enabled);
-	features = (MonoCPUFeatures) (features & ~mono_cpu_features_disabled);
-	return features;
+	return (features | mono_cpu_features_enabled) & ~mono_cpu_features_disabled;
```

Comment:
I think It was correct, it's just `features_` a redundant declaration, but thanks anyway!

Reply:
That was my first thought... then I realized that it would clobber `features` on subsequent calls which may not have been intended. I don't think it would ever misbehave but this looks cleaner anyway.

Reply:
`features` is not static.
This is cleaner imho. No need to repeat the left hand side of the assignment.